framework-cloud/framework-openshift-operator/src/main/java/org/kie/cloud/openshift/operator/deployment/WorkbenchOperatorDeployment.java

```diff
@@ -60,4 +60,23 @@ public void waitForScale() {
             RouterUtil.waitForRouter(getUrl());
         }
     }
+
+    @Override
+    public void changePassword(String newPassword) {
+        if (isReady()) {
+            KieApp kieApp = kieAppClient.withName(OpenShiftConstants.getKieApplicationName()).get();
+            kieApp.getSpec().getCommonConfig().setAdminPassword(newPassword);
+            kieAppClient.createOrReplace(kieApp);
+        }
+    }
+
+    @Override
+    public void changeUsernameAndPassword(String newUsername, String newPassword) {
+        if (isReady()) {
+            KieApp kieApp = kieAppClient.withName(OpenShiftConstants.getKieApplicationName()).get();
+            kieApp.getSpec().getCommonConfig().setAdminUser(newUsername);
+            kieApp.getSpec().getCommonConfig().setAdminPassword(newPassword);
+            kieAppClient.createOrReplace(kieApp);
+        }
+    }
```

Comment:
I think this is a leftover. If we remove the changes in HttpsWorkbenchTestProvider, this is not used anywhere.